### PR TITLE
If reparse fails return failure to host, in test build fail fast

### PR DESCRIPTION
**Pull request by agarwal-sandeep**

### CODE REVIEW

**Comment by agarwal-sandeep** [comment]
@akroshg please take a look

File: lib/Runtime/Debug/DebugContext.cpp [hunk 1]
```diff
@@ -260,7 +260,7 @@ namespace Js
             return false;
         });
 
-        if (this->scriptContext != nullptr && !this->scriptContext->IsClosed())
+        if (SUCCEEDED(hr) && this->scriptContext != nullptr && !this->scriptContext->IsClosed())
```
Comment (akroshg) [inline lib/Runtime/Debug/DebugContext.cpp]: So what is the idea now? if we fail on one we return the failures to host? but we have changed our scriptcontext already - so this is more unstable.

**Comment by agarwal-sandeep** [comment]
Host should fail fast. Or I can do fail fast in retail build as well.

**Comment by agarwal-sandeep** [comment]
This change looks risky. Shouldn't target release/1.2 will create a PR for master for future release.

**Pull request closed without merging**